packages/api/test/e2e/promise-queries.spec.ts

```diff
@@ -177,20 +177,197 @@ describe.skip('Promise e2e queries', () => {
     );
   });
 
-  it('queries state using double map key', async () => {
-    // TODO Update ['any', '0x1234'] to the key of a known event topic and update '[]' to the expected value
-    const eventTopics = await api.query.system.eventTopics(['any', '0x1234']);
+  describe('with plain type', async () => {
+    const TOTAL_ISSUANCE = 12582912;
```

Comment:
Bit sceptical about using this for dev chains, but it certainly matches with a fresh DB. Maybe we should rather use something simpler for dev, i.e. `balances.existentialDeposit`, i.e. `500`? (This should be a relative easy replace for all tests)

Reply:
Sorry, forget to fix this

Reply:
just updated test cases to use `balances.existentialDeposit` for plain type